.travis.yml

```diff
@@ -61,7 +61,10 @@ matrix:
       - npm i eslint-plugin-ember@latest --save-dev
       - npm config set spin false
 
-
     script:
       - npm run lint:js
       - npm test
+
+    after_success:
+      - chmod 777 ./scripts/gh_deploy.sh
```

Comment:
Run `git chmod 777 ./scripts/gh_deploy.sh` that will change the access parameters for the file. Then add it and commit. git will retain file permissions. And then we can remove this safely.

Reply:
@abishekvashok  `chmod ` is not a git command. You can update it by `git update-index --chmod=+x ./scripts/gh_deploy.sh` .